modules/google-analytics/wp-google-analytics.php

```diff
@@ -0,0 +1,615 @@
+	 */
+	public function admin_menu() {
+		add_options_page(__('Google Analytics', 'wp-google-analytics'), __('Google Analytics', 'wp-google-analytics'), 'manage_options', self::$page_slug, array( $this, 'settings_view' ) );
+	}
+
+	/**
+	 * Register our settings
+	 */
+	public function admin_init() {
+
+		register_setting( 'wga', 'wga', array( $this, 'sanitize_general_options' ) );
+
+		add_settings_section( 'wga_general', false, '__return_false', 'wga' );
+		add_settings_field( 'code', __( 'Google Analytics tracking ID:', 'wp-google-analytics' ), array( $this, 'field_code' ), 'wga', 'wga_general' );
+		add_settings_field( 'additional_items', __( 'Additional items to log:', 'wp-google-analytics' ), array( $this, 'field_additional_items' ), 'wga', 'wga_general' );
+		add_settings_field( 'do_not_track', __( 'Visits to ignore:', 'wp-google-analytics' ), array( $this, 'field_do_not_track' ), 'wga', 'wga_general' );
+		add_settings_field( 'other_options', __( 'Other options:', 'wp-google-analytics' ), array( $this, 'field_other_options' ), 'wga', 'wga_general' );
+		add_settings_field( 'custom_vars', __( 'Custom variables:', 'wp-google-analytics' ), array( $this, 'field_custom_variables' ), 'wga', 'wga_general' );
+	}
+
+	/**
+	 * Where the user adds their Google Analytics code
+	 */
+	public function field_code() {
+		// Display the tokens in the right column of the page
+		echo '<div id="tokens-description" style="position:absolute;margin-left:600px;margin-right:50px;">';
+		echo '<span>' . __( 'Use tokens in your custom variables to make your fields dynamic based on context. Here are some of the tokens you can use:' ) . '</span>';
+		echo '<table style="text-align:left;">';
+		echo '<thead><tr><td>' . __( 'Token', 'wp-google-analytics' ) . '</td><td>' . __( 'Description', 'wp-google-analytics' ) . '</td><td>' . __( 'Return value', 'wp-google-analytics' ) . '</td></tr></thead>';
+		echo '<tbody>';
+		foreach( $this->tokens as $token ) {
+			echo '<tr>';
+			echo '<td>' . esc_html( $token['token'] ) . '</td>';
+			echo '<td>' . esc_html( $token['description'] ) . '</td>';
+			echo '<td>' . esc_html( $token['retval'] ) . '</td>';
+			echo '</tr>';
+		}
+		echo '</tbody>';
+		echo '</table>';
+		echo '</div>';
+
+		echo '<input name="wga[code]" id="wga-code" type="text" value="' . esc_attr( $this->_get_options( 'code' ) ) . '" />';
+		echo '<p class="description">' . __( 'Paste your Google Analytics tracking ID (e.g. "UA-XXXXXX-X") into the field.', 'wp-google-analytics' ) . '</p>';
+	}
+
+	/**
+	 * Option to log additional items
+	 */
+	public function field_additional_items() {
+		$addtl_items = array(
+				'log_404s'       => __( 'Log 404 errors as events', 'wp-google-analytics' ),
+				'log_searches'   => sprintf( __( 'Log searches as /search/{search}?referrer={referrer} (<a href="%s">deprecated</a>)', 'wp-google-analytics' ), 'http://wordpress.org/extend/plugins/wp-google-analytics/faq/' ),
```

Comment:
Since it's bundled in Jetpack, we should probably link to a Jetpack support doc instead.